thorlcr/master/thgraphmanager.cpp

```diff
@@ -1138,6 +1139,11 @@ void abortThor(IException *e, unsigned errCode, bool abortCurrentJob)
         LOG(MCdebugProgress, thorJob, "abortThor called");
         if (jM)
             jM->stop();
+        if (thorQueue)
+        {
+            Owned<IJobQueue> queue = thorQueue.getClear();
```

Comment:
Why the getClear()?  I think that may create a potential race condition, getLink() is probably safer.

Reply:
agree, could cause problems, I'll change to getLink